### PR TITLE
Feature/ados76 personal info

### PR DESCRIPTION
**Pull request by Arochawms10**
Screen: Personal Info foreign

### CODE REVIEW

File: src/app/screens/PersonalInformationForeign/styles.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+  height: 50;
+  margin-top: 30;
+  margin-bottom: 18;
+`;
+
+export const ContainerButton = styled.View`
+  align-items: center;
+  height: 35;
+  width: 100;
+  margin: 0 auto;
```
Comment (jhonwms) [inline src/app/screens/PersonalInformationForeign/styles.js:58]: its necessary `margin: 0 auto?`

File: src/app/screens/PersonalInformationForeign/index.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+type Props = {
+  navigation: any,
+};
+const datePicker = async () => {
+  try {
+    await DatePickerAndroid.open({
+      date: new Date(),
+    });
+  } catch ({ code, message }) {
+    console.warn('Cannot open date picker', message);
```
Comment (usaqui-albert) [inline src/app/screens/PersonalInformationForeign/index.js:25]: Let's avoid `console.warn`. Set a TODO message to handle this event.

File: src/app/screens/PersonalInformationForeign/styles.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+export const TextSmall = styled.Text`
+  font-weight: normal;
+  color: ${props => props.theme.colors.white};
+  align-self: center;
+`;
+export const IconSize = styled.Image`
+  width: 30;
+  height: 30;
+  margin-vertical: 5;
+  z-index: 1;
```
Comment (usaqui-albert) [inline src/app/screens/PersonalInformationForeign/styles.js:24]: Remember to use `z-index` 10 by 10

File: src/app/screens/PersonalInformationForeign/index.js [hunk 4]
```diff
@@ -0,0 +1,68 @@
+// @flow
+import React from 'react';
+import LinearGradientContainer from '@ui/LinearGradientContainer';
+import TextInput from '@ui/TextInput';
+import IconResident from '@images/tag-icon.png';
+import { ScrollView, DatePickerAndroid } from 'react-native';
+import { ButtomButtomFixed } from '../../components/hocs/bottomButtomFixed';
```
Comment (usaqui-albert) [inline src/app/screens/PersonalInformationForeign/index.js:7]: Just asking, what about the `@hocs` alias?

**Pull request merged**